src/site/content/en/progressive-web-apps/progressively-enhance-your-pwa/resize.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+set -e
+maxwidth="1600"  # in pixels, the widest image you want to allow.
+
+#find all .jpg in current dir and subdirectories
+FILES="$(find . -iname '*.png')"
+
+for imagefile in $FILES
+do
+if [ -f "$imagefile" ]; then
+imgwidth=`sips --getProperty pixelWidth "$imagefile" | awk '/pixelWidth/ {print $2}'`
+else
+    echo "Oops, "$imagefile" does not exist." 
+    exit
+fi
+
+if [ $imgwidth -gt $maxwidth ]; then
+    echo " - Image too big. Resizing..."
+    sips --resampleWidth $maxwidth "$imagefile" > /dev/null 2>&1  # to hide sips' ugly output 
+    imgwidth=`sips --getProperty pixelWidth "$imagefile" | awk '/pixelWidth/ {print $2}'`
+    imgheight=`sips --getProperty pixelHeight "$imagefile" | awk '/pixelHeight/ {print $2}'`
+    echo " - Resized "$imagefile" to $imgwidth""px wide by $imgheight""px tall";
+fi
+done
```

Comment:
Please delete this image resizing script

* It shouldn't live in the content directory
* I'm not sure if Rob or anyone else will use it so I'm not sure if it should live in the repo yet (we would have to talk to engineering about that)
* If you want to have a reference to it, you can create a Gist and link to the Gist via a comment in your source code

Reply:
In other words, it could potentially be useful to have in the repo but we should discuss that in a separate PR

Reply:
Oops, sorry. The second time that I accidentally commit this… It could probably be part of a GitHub action or something.